src/component/User/Profile.js

```diff
@@ -18,39 +24,73 @@ import {
 import style from './Profile.module.css';
 
 const Profile = () => {
-  // const handleLogout = () => {
-  //   dispatch({ type: 'LOGOUT' });
-  // };
+  const [errorMessage, setErrorMessage] = useState('');
+  const [successMessage, setSuccessMessage] = useState('');
+  const [state, dispatch] = useReducer(userReducer, initialState);
+  const [logaut, setLogaut] = useState(false);
```

Comment:
Here is a typo here: it should be `logout`, `setLogout`. Please change it across the file.